CHARTER.md

```diff
@@ -1,107 +1,121 @@
+This Technical Charter sets forth the responsibilities and procedures for technical contribution to, and oversight of, the ORBIT open source community, which has been established as a Working Group (the "WG") under the Open Source Security Foundation (the “OpenSSF”).  All contributors (including committers, maintainers, and other technical positions) and other participants in the WG (collectively, “Collaborators”) must comply with the terms of this Technical Charter and the OpenSSF Charter.
 
-#### 1. Mission and Scope of the Technical Initiative
+## 1. Mission and Scope
 
-- a. The mission of the Technical Initiative is to [MISSION STATEMENT].
+- a. The mission of the WG is to develop and maintain interoperable resources related to the identification and presentation of security-relevant data.
 
-- b. The scope of the Technical Initiative includes collaborative development under the Technical Initiative License (as defined herein) supporting the mission, including organizing collaboration activities, defining best practices, documentation, testing, integration, and the creation of other artifacts that support the mission.
+- b. The scope of the WG includes collaborative development under the WG License (as defined herein) supporting the mission, including organizing collaboration activities, defining best practices, documentation, testing, integration, and the creation of other artifacts that support the mission.
 
-#### 2. Technical Steering Committee
+- c. Participation in the WG through attendance or contribution is open to anyone, whether a OpenSSF member or not, so long as they abide by the terms of this Technical Charter.
 
-- a. The Technical Steering Committee (the "TSC") will be responsible for all oversight of the Technical Initiative. 
+## 2. Technical Steering Committee
 
-- b. The TSC voting members are initially the Technical Initiative’s Maintainers. The Maintainers will be documented in the Technical Initiative repository. The TSC is responsible for determining the future process for defining voting members of the TSC, and any such alternative approach will also be documented appropriately.  Any meetings of the Technical Steering Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person. 
+- a. The Technical Steering Committee (the "TSC") will be responsible for all oversight of the WG and reporting to the OpenSSF’s Technical Advisory Council (the "TAC").
 
-- c. The Technical Initiative generally will involve Collaborators and Contributors. The TSC may adopt or modify additional roles so long as the roles are documented in the Technical Initiative’s repository. Unless otherwise documented: 
+- b. The TSC is composed of a single lead from each Technical Initiative in the WG, documented in the charter of the respective Technical Initiative.
```

Comment:
If there are multiple leads for a TI, should the TSC representative of a TI within the WG be selected by consensus of those leads? We can ignore this comment if that's not the case for any TI.

Reply:
The intent is to have a single representative from each, regardless of how the TI organizes itself.